### PR TITLE
refactor(breadcrumb): refactor the type definition of breadcrumb

### PR DESCRIPTION
**Pull request by qige2016**
Please make sure these boxes are checked before submitting your PR, thank you!

* [x] Make sure you follow Element's contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
* [x] Make sure you are merging your commits to `dev` branch.
* [x] Add some descriptions and refer to relative issues for your PR.

### CODE REVIEW

File: packages/breadcrumb/src/index.vue [hunk 1]
```diff
@@ -11,11 +11,7 @@
 
 <script lang="ts">
 import { defineComponent, provide, ref, onMounted } from 'vue'
-
-interface IBreadcrumbProps {
-  separator: string
-  separatorClass: string
-}
+import { IBreadcrumbProps } from './breadcrumb'
```
Comment (JeremyWuuuuu) [inline packages/breadcrumb/src/index.vue:14]: Please use `import type { IBreadcrumbProps }`

Comment (qige2016) [reply packages/breadcrumb/src/index.vue]: Fixed.

Comment (JeremyWuuuuu) [reply packages/breadcrumb/src/index.vue]: Next time you can resolve this conversation after you addressed the comment ;)

**Review by jw-foss — APPROVED** [review]

File: packages/breadcrumb/src/index.vue (hunk 1 above)
Comment (qige2016) [reply packages/breadcrumb/src/index.vue]: Ok.

**Pull request merged**